docs/ReleaseNotes.md

```diff
@@ -23,7 +23,7 @@ The Guava dependency version has been updated to 31.1. Projects may need to chec
 * **Performance** Improvement 1 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Performance** Improvement 2 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
 * **Performance** Improvement 3 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
-* **Performance** Improvement 4 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
+* **Performance** Single record deletes now issue multiple single-key clears instead of one range clear, which should be an improvement for LSM-based FDB storage engines [(Issue #1493)](https://github.com/FoundationDB/fdb-record-layer/issues/1493)
```

Comment:
The issue describes: 
> Note: at the FDB protocol level, there actually isn't a "single key clear" and all clears are range clears. The "clear key" operation is translated into a range clear over a single key range (i.e., to clear a single key, the range starting at that key and ending at that key plus \0 is cleared) at the protocol level. So, for any storage engine to somehow optimize the single key range clear case, it would need to either change the protocol to introduce a separate ClearKey mutation type or translate the range back into its original single key.

Has that situation changed, or is this actually going to result in more range deletes on the storage engine, making things worse?

Reply:
The RocksDB storage engine recognizes single key range clears: https://github.com/apple/foundationdb/blob/main/fdbserver/KeyValueStoreRocksDB.actor.cpp#L1865-L1866.